--- a/kibana-daemon.rb
+++ b/kibana-daemon.rb
@@ -1,6 +1,6 @@
 #!/usr/bin/env ruby
 
-require 'rubygems'
+#require 'rubygems'
 require 'daemons'
 require 'pathname'
 require 'fileutils'
--- a/kibana.rb
+++ b/kibana.rb
@@ -1,4 +1,4 @@
-require 'rubygems'
+#require 'rubygems'
 require 'sinatra'
 require 'json'
 require 'time'
--- a/lib/client_request.rb
+++ b/lib/client_request.rb
@@ -1,4 +1,4 @@
-require 'rubygems'
+#require 'rubygems'
 require 'json'
 require 'base64'
 require 'time'
--- a/lib/kelastic.rb
+++ b/lib/kelastic.rb
@@ -1,4 +1,4 @@
-require 'rubygems'
+#require 'rubygems'
 require 'json'
 require 'time'
 require 'net/http'
--- a/lib/query.rb
+++ b/lib/query.rb
@@ -1,4 +1,4 @@
-require 'rubygems'
+#require 'rubygems'
 require 'json'
 require 'tzinfo'
 
